packages/core/admin/admin/src/content-manager/pages/App.tsx

```diff
@@ -239,26 +226,21 @@ const selectSchemas = createSelector(
 const reducer = (state: ContentManagerAppState = initialState, action: Action) =>
   produce(state, (draftState) => {
     switch (action.type) {
-      case GET_INIT_DATA: {
-        draftState.status = 'loading';
-        break;
-      }
```

Comment:
annoyingly i think this is a breaking change since this exists in redux _someone_ might be relying on it in a middleware or subscriber. I would add a todo to remove it in V5 though because it's a bit unnecessary.